apps/recovery-portal/src/main/webapp/password-reset.jsp

```diff
@@ -163,13 +163,13 @@
                             %>
 
                            <div class="field">
-                                <label for="reset-password2">
+                                <label for="confirm-password">
                                     <%=IdentityManagementEndpointUtil.i18n(recoveryResourceBundle, "Confirm.password")%>
                                 </label>
                                 <div class="ui right icon input">
                                     <input
-                                        id="reset-password2"
-                                        name="reset-password2"
+                                        id="confirm-password"
```

Comment:
Won't we need to change line 224 too with this change?

Reply:
adjusted with 497ebab76c90993725dc62f65153ab2f3b247d2e.

manually tested flow, works as expected